src/app/core/rpc/rpx.service.ts

```diff
@@ -65,5 +65,14 @@ export class RPXService {
     });
   }
 
+  runNotification(channel: string, ...args: any[]): Observable<any> {
```

Comment:
Does this work? Variable channel names should not be allowed.
Also rpx class is not the right place to put this.

Reply:
Variable channel names should not be allowed , yes we can change it the exact name what it is using :)